linux-bore/PKGBUILD

```diff
@@ -822,16 +823,16 @@ _package-nvidia(){
 pkgname=("$pkgbase" "$pkgbase-headers")
 [ -n "$_build_zfs" ] && pkgname+=("$pkgbase-zfs")
 [ -n "$_build_nvidia" ] && pkgname+=("$pkgbase-nvidia")
+[ "$_cpusched" = "sched-ext" ] && pkgname+=("$pkgbase-scheduler")
```

Comment:
I think the scheduler suffix might be a little confusing

Reply:
Maybe adding a "s"?

Actually these are the schedulers compiled for sched-ext

Reply:
Maybe just add the ``sched-ext-tools`` suffix? I think that would make more sense.

Reply:
Tools sounds weird in my eyes.
This "kernel" is anyways just a playground to give and bring more support into sched-ext testing and developing. They are searching for more benchmarks and testers with "scx_rusty" and others for gaming, so we want to support there as CachyOS.

I think schedulers sounds good compared to "tools". It basically packages the "schedulers" for sched-ext and not any tools, in my eyes.
Naming is complicated x)